### ISSUE TITLE
Tor connection not working on version 2.0.8

### ISSUE THREAD

**Issue by amcitaly** [Blockstream/green_qt#133]
### Description

Can't connect anymore via my Umbrel node (tor)

### Version
2.0.8

The wallet keep on "connecting" instead start

### Screenshots

![green umbrel server](https://github.com/user-attachments/assets/c5051c92-81b6-4d7c-b911-90af714b8c70)
![keep connecting](https://github.com/user-attachments/assets/2206acb3-dcbf-483a-8f3b-59496b3bbc4d)
![green 2 0 6](https://github.com/user-attachments/assets/d5589c45-5a96-4d54-9682-fcb4956dc32b)

### Device or machine
Intel mini pc

Linux Kubuntu

### Additional info
I keep also the version 2.0.6, same settings and connect without issue, this happen just with this new version

**Comment by borgbyte** [comment]
Please test again, there were some server issues.

**Comment by amcitaly** [comment]
Yes it connect to my Umbrel node, no, is not working also in version 2.0.9 with "Enable TLS/SSL" on. Instead on version 2.0.6 always no problem, it connect very fast.

**Comment by borgbyte** [comment]
Can you try 2.0.10?

**Comment by amcitaly** [comment]
![connection failed](https://github.com/user-attachments/assets/2cc9e85f-4aa6-44c1-8552-a2a640c3da12)

No, I double check the settings are the same, also because if I change on version 2.0.6 that works, the changes are reflected on other version and vice versa

**Comment by amcitaly** [comment]
2.0.12 Still not working
![Green tor connection 2 0 12](https://github.com/user-attachments/assets/0a7b1e8d-e2a4-4bbf-b361-8dbf26d0230e)

**Comment by ardier16** [comment]
@amcitaly Thanks for the detailed testing and comparisons with version 2.0.6.

The Tor and custom Electrum connection handling has changed since this issue was reported. Could you please test again using the [latest Linux AppImage](https://github.com/Blockstream/green_qt/releases/latest)?

Please use the current Electrum endpoint and port shown by Umbrel, and ensure **Enable TLS/SSL** matches the connection type provided by your server.

If it still fails, please provide:

- Blockstream app version
- Kubuntu version
- Umbrel version
- Electrum server application and version
- Whether TLS/SSL is enabled
- The connection error shown by the app

Please do not post your `.onion` address or other private server details publicly. You can use **Get Support** in the app with **Share logs** enabled and mention GitHub issue `#133`.

**Comment by amcitaly** [comment]
Hello @ardier16 is working now! And is very fast to connect. Last version 3.4.1

**Comment by ardier16** [comment]
@amcitaly glad to hear! Sorry for the late response and thank you for patience! I'm closing this issue as completed.
